### ISSUE TITLE
Feature request - support multiple accounts

### ISSUE THREAD

**Issue by Gargo** [Beelzebub2/AutoBanana#24]
For example in banana clicker you get in-game drops quickly and you could switch to another account to earn even more drops

**Comment by Beelzebub2** [comment]
I'll try to get into that when i have time today

**Comment by Beelzebub2** [comment]
@Gargo it's done test it out

**Comment by Gargo** [comment]
@Beelzebub2 sorry for delay but it works strange - it allows you to switch between accounts but manually. It could be done automatically, possibly by specifying a list of logins and passwords.
I've already seen 2 other repos (one runs games and another one switches between accounts) but I don't know yet how to combine them

**Comment by Beelzebub2** [comment]
In order for it to work you need to login to both or all accounts and tick the remember me checkbox after that it should work

**Comment by Beelzebub2** [comment]
Steam saves the login info to a vdf file (encrypted i belive and sure hope so) all i do is change the user that's logged in, in the registry steam reads that and if it has the loggin info on the autologin goes straight to that account.

**Comment by Beelzebub2** [comment]
@Gargo Try logging in to the accounts ticking the remember me check box and come back with the feedback. If it still doesn't work I'll try to find a workaround.

**Comment by Gargo** [comment]
@Beelzebub2 tried, it doesn't work. With "Remember me" it just shows the window "choose account"

**Comment by Gargo** [comment]
https://github.com/sw2719/steam-account-switcher
this may help with account switching

**Comment by Beelzebub2** [comment]
> @Beelzebub2 tried, it doesn't work. With "Remember me" it just shows the window "choose account"

Oh yeah i forgot you gotta turn that off

**Comment by Gargo** [comment]
@Beelzebub2 "Remember me" just allows to skip entering password but I still need to choose it manually. Or did you mean something else?

**Comment by Beelzebub2** [comment]
> @Beelzebub2 "Remember me" just allows to skip entering password but I still need to choose it manually. Or did you mean something else?

That Account selection PopUp is something you can turn off in steam

**Comment by Gargo** [comment]
@Beelzebub2 I found this option - something like "Ask which account to use on each steam launch" but it is already disabled.
And it seems this option isn't related to the problem. In my case this window appears because i logged in 2 accounts but chose no one from them.

**Comment by Beelzebub2** [comment]
> @Beelzebub2 I found this option - something like "Ask which account to use on each steam launch" but it is already disabled. And it seems this option isn't related to the problem. In my case this window appears because i logged in 2 accounts but chose no one from them.

Damm, alright I'll look into it when i have the time

**Comment by Beelzebub2** [comment]
> Hello, what about account changing problem?

test it out

**Comment by SudoBlocks** [comment]
> > Hello, what about account changing problem?
> 
> test it out

thx! also it have one big problem - auto login of steam accounts are reset, can u fix it? also wanna see a total timer in config.

**Comment by SudoBlocks** [comment]
Yes, it shows, everything worked fine, after reboot everything showed also fine, but only after the first pass, then errors and reset of authorizations in fact. In total 1 account was save, others is delete

**Comment by Beelzebub2** [comment]
> Yes, it shows, everything worked fine, after reboot everything showed also fine, but only after the first pass, then errors and reset of authorizations in fact. In total 1 account was save, others is delete

Add me on discord beelzebub2. I'll help tomorrow
